api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -194,7 +194,8 @@ public static void loadModules(List<File> modulesToLoad) {
 					try {
 						Module mod = loadModule(f, true); // last module loaded wins
 						log.debug("Loaded module: " + mod + " successfully");
-					} catch (Exception e) {
+					}
```

Comment:
@teleivo i created a new pull request as i messed up the git commit history with sqash to much and push to early. still learning git,
i don't see  where i can remove this pull request but will try further.